pymc_marketing/mmm/lift_test.py

```diff
@@ -287,105 +284,6 @@ def saturation_function(x, alpha, lam):
     )
 
 
-def add_menten_empirical_lift_measurements_to_likelihood(
```

Comment:
Where did yo move the lift test functions to?

Reply:
They aren't needed anymore.

Lift tests are supported automatically based on the inheriting from `SaturationTransformation`

Reply:
cool!